### ISSUE TITLE
Improve Add trip button

### ISSUE THREAD

**Issue by Swati4star** [project-travel-mate/Travel-Mate#158]
Currently, the trips page look like this:
![trips](https://user-images.githubusercontent.com/9164477/42121760-c5236f08-7c52-11e8-898e-3b344495e6c2.png)

- Remove the add trip button from here & add a floating action button ( https://developer.android.com/guide/topics/ui/floating-action-button ) at the bottom of this page
- On clicking that button, the page for add trip should open up.
- In case, the user doesn't have any trips, show a text view with this text "You currently don't have any trips. Click on + button to add a trip and add friends to it"

**Comment by brijeshshah13** [comment]
@Swati4star M taking up this one.

**Comment by brijeshshah13** [comment]
@Swati4star After pulling the latest changes, when I click on the existing `Add New Trip` button, the app crashes. Please verify.

**Comment by Swati4star** [comment]
@brijeshshah13 I tried out with the latest code only. I am not facing this crash.

**Comment by Swati4star** [comment]
@brijeshshah13 Is the issue solved?

**Comment by brijeshshah13** [comment]
@Swati4star Nope. I'm attaching the gif along with the log.

![20180703_110417](https://user-images.githubusercontent.com/18366144/42201497-cbc31542-7eb5-11e8-9e75-bb6e401bda5e.gif)

```
android.util.AndroidRuntimeException: Calling startActivity() from outside of an Activity  context requires the FLAG_ACTIVITY_NEW_TASK flag. Is this really what you want?
        at android.app.ContextImpl.startActivity(ContextImpl.java:677)
        at android.app.ContextImpl.startActivity(ContextImpl.java:664)
        at android.content.ContextWrapper.startActivity(ContextWrapper.java:331)
        at io.github.project_travel_mate.mytrips.MyTripsAdapter.lambda$getView$0$MyTripsAdapter(MyTripsAdapter.java:61)
        at io.github.project_travel_mate.mytrips.MyTripsAdapter$$Lambda$0.onClick(Unknown Source)
        at android.view.View.performClick(View.java:5268)
        at android.view.View$PerformClick.run(View.java:21550)
        at android.os.Handler.handleCallback(Handler.java:822)
        at android.os.Handler.dispatchMessage(Handler.java:104)
        at android.os.Looper.loop(Looper.java:207)
        at android.app.ActivityThread.main(ActivityThread.java:5811)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:791)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:681)
```

**Comment by Swati4star** [comment]
It shows the error is in line : https://github.com/project-travel-mate/Travel-Mate/blob/9ba9f0ce25bb308e33b1cd03b1f747a9c6148eb8/Android/app/src/main/java/io/github/project_travel_mate/mytrips/MyTripsAdapter.java#L61
Can you look what is the problem you are facing? Also, anyways you'll be removing the button from here, so this shouldn't stop you from fixing the issue. @brijeshshah13

**Comment by Swati4star** [comment]
@brijeshshah13 Status?

**Comment by Swati4star** [comment]
Fixed by ab17841e06dcc195e5331b6d62349af85e80d6f7
